pandas/tests/extension/test_arrow.py

```diff
@@ -1406,3 +1406,20 @@ def test_astype_from_non_pyarrow(data):
     assert not isinstance(pd_array.dtype, ArrowDtype)
     assert isinstance(result.dtype, ArrowDtype)
     tm.assert_extension_array_equal(result, data)
+
+
+def test_to_numpy_with_defaults(data, request):
```

Comment:
Nit: Looks like `request` isn't needed anymore

Reply:
removed it. thanks